commit b261dab8ed8c66101e4506f4f68f7743bf24667e
Author: brummer10 <brummer-@web.de>
Date:   2024-08-30 19:46:52 +0200

    Fix issue #25, Error when compiling for AArch64: unknown architecture armv8

diff --git a/Ratatouille/Ratatouille.cpp b/Ratatouille/Ratatouille.cpp
index 01083e5..e8e8b6c 100644
--- a/Ratatouille/Ratatouille.cpp
+++ b/Ratatouille/Ratatouille.cpp
@@ -23,13 +23,13 @@
 #include <lv2/core/lv2.h>
 #include <lv2/atom/atom.h>
 #include <lv2/atom/util.h>
-#include "lv2/atom/forge.h"
+#include <lv2/atom/forge.h>
 #include <lv2/midi/midi.h>
 #include <lv2/urid/urid.h>
-#include "lv2/patch/patch.h"
-#include "lv2/options/options.h"
-#include "lv2/state/state.h"
-#include "lv2/worker/worker.h"
+#include <lv2/patch/patch.h>
+#include <lv2/options/options.h>
+#include <lv2/state/state.h>
+#include <lv2/worker/worker.h>
 #include <lv2/buf-size/buf-size.h>
 
 ///////////////////////// MACRO SUPPORT ////////////////////////////////
diff --git a/Ratatouille/makefile b/Ratatouille/makefile
index 241f580..29d455d 100644
--- a/Ratatouille/makefile
+++ b/Ratatouille/makefile
@@ -77,7 +77,7 @@ ifneq ($(MAKECMDGOALS),install)
         $(info $(yellow) INFO: $(reset)optimised for $(blue)$(SSE_CFLAGS)$(reset))
       endif
     else ifeq ($(shell arch), aarch64)
-      CXXFLAGS += -march=armv8
+      CXXFLAGS += -march=armv8-a
       $(info $(yellow) INFO: $(reset)optimised for $(blue)-march=armv8 $(SSE_CFLAGS)$(reset))
     endif
   endif
